common/org.testeditor.dsl.common.ui.tests/src/org/testeditor/dsl/common/ui/utils/ProjectContentGeneratorTest.xtend

```diff
@@ -157,6 +159,7 @@ class ProjectContentGeneratorTest extends AbstractTest {
 	@Test
 	def void testGetVersion() {
 		// given (this is a plugin test)
+		Assume.assumeTrue(isWithinOSGIContext)
```

Comment:
Static import of what exactly? I'm not sure what you mean. Could you please write some more on this?